src/fsharp/fscopts.fs

```diff
@@ -444,7 +444,6 @@ let testFlag tcConfigB =
 // not shown in fsc.exe help, no warning on use, motiviation is for use from VS
 let vsSpecificFlags (tcConfigB: TcConfigBuilder) = 
   [ CompilerOption("vserrors", tagNone, OptionUnit (fun () -> tcConfigB.errorStyle <- ErrorStyle.VSErrors), None, None);
-    CompilerOption("validate-type-providers", tagNone, OptionUnit (fun () -> tcConfigB.validateTypeProviders <- true), None, None);
```

Comment:
I do wonder if this flag should be left in and ignored.  

In theory the next version of the Visual F# fsc.exe compiler will only be used with the corresponding Visual F# Tools and FSharp.Build.dll.  But perhaps somehow the compiler will be invoked via the old tools, or an open source version of the tools, or some other tool that has copied out the command-line settings from the Visual F# Tools invocations of fsc.exe (which includes this flag).  Perhaps just ignoring the option would be better.